paimon-hive/paimon-hive-catalog/src/main/java/org/apache/paimon/hive/HiveCatalog.java

```diff
@@ -334,6 +335,8 @@ public void alterTable(
             // first commit changes to underlying files
             TableSchema schema = schemaManager.commitChanges(changes);
```

Comment:
validate in `schemaManager.commitChanges`?
assert in `AddColumn` and `RenameColumn`.

Otherwise, the upper case fields will be commit into file system.

Reply:
Will it affect other types of catalogs

Reply:
checkFieldNamesUpperCaseInSchemaChange before commit to underlying files . please take a look